constitution/prachtsaal-constitution.md

```diff
@@ -383,6 +383,7 @@ j. to inspect the summarized result of the audit report.
 
 a. the General Assembly (Generalversammlung),
 b. the Board of Directors (Vorstand)
+c. the Authorized Representative (Bevollmächtigter)
```

Comment:
This phrasing is intended. the Bevollmächtigter, does not constitute a cooperative body, however once we have more than 20 members, we will have to put the Supervisory Board here, which is a cooperative body. This structure is taken from constitution templates for small cooperatives.